server/index.ts

```diff
@@ -17,6 +17,25 @@ import { schema, TypeOf } from '@osd/config-schema';
 import { PluginInitializerContext, PluginConfigDescriptor } from '../../../src/core/server';
 import { SecurityPlugin } from './plugin';
 
+const validateAuthType = (value: string[]) => {
+  const supportedAuthTypes = [
+    '',
+    'basicauth',
+    'jwt',
+    'openid',
+    'saml',
+    'proxy',
+    'kerberos',
+    'proxycache',
+  ];
```

Comment:
AFAIK, Kerberos has not been supported by this plugin yet. Should we remove it for now?
Does this feature enabled all these auth types? or only Basic Auth, SAML and OIDC?
If it does not enable all these auth types, but all the supported auth types are included here, so I guess you expect this method is also called in somewhere else, right?

Reply:
@cliu123 This is generic authentication type checking from configuration schema level. So it covers all the authentication types supported by the plugin. Multiple authentication handler will have another supported authentication type checking, which will only allow Basic, SAML and OIDC.